frontend/src/core/app.js

```diff
@@ -54,6 +55,7 @@
   * Start app load
   */
   loadLibraries(function() {
+    window.Handlebars = $.extend(require('handlebars'), window.Handlebars);
```

Comment:
Can this go in `loadLibraries` before the `callback` param is called? (like we do with stuff in `loadCore`)